docs/assets/js/toggle-details.js

```diff
@@ -1,16 +1,30 @@
 const HIDDEN_CLASS = 'u-hidden';
-export const TOGGLE_ATTRIBUTE = 'data-toggle-details';
+const TOGGLE_ATTRIBUTE = 'data-toggle-details';
+const STATE_SHOW = 'show';
+const STATE_HIDE = 'hide';
+
+let isShowingAllDetails = true;
```

Comment:
The name of this variable is a little confusing. On page load aren't the details *not* being shown meaning this should start as `false`?

Reply:
It's true here because I set it equal to its inverse when the toggle function [is called](https://github.com/cfpb/design-system/pull/819/files#diff-3afedd31e202de2df455d7511ae78262R43). I suppose I could move where its set to the end of the function though.

Reply:
Updated in https://github.com/cfpb/design-system/pull/819/commits/2eaa81e1afe0c35f74677970565af48170a597d1